baselines/feddebug/feddebug/conf/base.yaml

```diff
@@ -0,0 +1,92 @@
+device: cpu
+total_gpus: 1
+total_cpus: 10
+client_cpus: 1
+client_gpu: 0.2
```

Comment:
Could we rename this to `num_cpus` and `num_gpus`? that adheres better with the syntax used internally (as you do in `utils.config_sim_resources()`). Alternatively you could put both under the same `client_resources` tag (as done in other baselines):
```yaml
client_resources:
    num_cpus: 1
    num_gpus: 0.2
```

Reply:
resolved